src/generators.jl

```diff
@@ -48,8 +46,7 @@ julia> permutation_vector_pop(2, 5, ["a", "b", "c", "d", "e"]; replacement=false
 ```
 """
 function permutation_vector_pop(n, d, pool; replacement=false, rng=Random.GLOBAL_RNG)
-    thepop = [sample(rng, pool, d, replace=replacement, ordered=false) for i in 1:n]
-    return thepop
+    [sample(rng, pool, d, replace=replacement, ordered=false) for _ in 1:n]
```

Comment:
I'd prefer to add the return keyword even if it's not really necessary. It's both part of the guidelines and in a sense easy to read. So  `return [sample(rng, pool, d, replace=replacement, ordered=false) for _ in 1:n]` would be great